jobs/build-webapp.groovy

```diff
@@ -505,6 +505,12 @@ def uploadGraphqlSafelist() {
    // the new version.  This depends on the static deploy having
    // finished, since that updates the safelist.
    echo("Pre-computing the query-plans for the latest safelist queries.");
+   echo("NOTE: this command will give a lot of errors like");
+   echo("- Error getting query plan");
+   echo("- query did not validate against schema");
+   echo("This is expected, since some obsolete queries in the safelist do");
+   echo("not validate against the current schema.");
+   echo("These errors do NOT indicate a problem with the deploy.");
```

Comment:
**question:** Will there ever be errors that look like this that DO mean a problem with the deploy? If so, how will folks differentiate? Can we just not log the ones that aren't a problem if they aren't valuable information?

Reply:
> Will there ever be errors that look like this that DO mean a problem with the deploy?

If there were, it would have shown up in unittests.